src/app/components/core/notifications/notification-modal/notification-modal.component.html

```diff
@@ -3,8 +3,8 @@
   <div class="modal-card">
     <header class="modal-card-head">
       <span *ngIf="notification" class="modal-card-title title-name">{{notification.name}}
-        <span><i class="bi bi-info-circle plugin-info" aria-hidden="true" [title]="'Rule Plugin: ' + notification.rule + '&#013;Action Plugin: ' + notification.channel"></i></span>
-        <p class="help has-text-grey ctrl-pipeline">Notification Instance</p>
```

Comment:
This ctrl-pipeline class css can be removed, or used somewhere?!

Reply:
It is being used in `add-pipeline-filter`. I've removed its unwanted occurrences.

Reply:
OK